tests/functional/gaze_file_processing_test.py

```diff
@@ -84,6 +84,7 @@ def fixture_gaze_init_kwargs(request):
             'time_unit': pm.datasets.SBSAT().time_unit,
             'pixel_columns': pm.datasets.SBSAT().pixel_columns,
             'experiment': pm.datasets.SBSAT().experiment,
+            'trial_columns': pm.datasets.SBSAT().trial_columns,
```

Comment:
Had to know about the trial columns here as we have a string type trial column in this dataset. GazeDataFrame needs to exclude trial columns for processing in `transforms.resample`. If not doing so interpolation would fail here. (Can not interpolate string)

Reply:
Is it by accident that all `trial_columns` attributes are missing here for the other dataset?